client/router.js

```diff
@@ -364,7 +369,13 @@ Router.prototype.initialize = function(options) {
   // in unpredicatable manner. See #168
   // this is the default behaviour and we need keep it like that
   // we are doing a hack. see .path()
-  this._page({decodeURLComponents: true, hashbang: !!options.hashbang});
+  this._page({
+    basePath: this._basePath,
```

Comment:
I'm not sure if this is useful, the below call to `this._page.base` since to be enough on my browser but maybe `basePath` is used for compatibility with old IE?